server/app/routes/repos/routes.repos.putActive.js

```diff
@@ -2,17 +2,28 @@
 var router = require('express').Router();
 var payloadParser = require('../../github-data/parsers')
 var Promise = require('bluebird')
+var Griller = require('../../../middleware/github-griller');
 
 var Board = require('mongoose').model('Board');
 
 module.exports = router;
 
+
 router.put('/:boardID/active', function(req, res, next) {
-	var github = req.user.githubAccess;
+  return Griller(req).getOneRepo(req.params.boardID)
+  .then(function(repo) {
+    console.log('----------repo', repo)
+    console.log("here is the req", req)
+  })
 
```

Comment:
@ssarda77 I've taken over this route for now - instead of marking this one repo as active, I'm taking the opportunity to getOneRepo.  Before merging in this branch, I'll make sure to set this branch so that it doesn't break current functionality.